php-files/index.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$db = new PDO('mysql:host=192.168.20.20; dbname=fish-finder', 'root', '');
+
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$sql = $db->prepare('SELECT * FROM `fish`;');
```

Comment:
SELECTing * is bad practice as it makes the query very slow, it's better to be specific as to which fields you want to SELECT